ui/src/pages/Create.js

```diff
@@ -477,13 +477,32 @@ class Create extends Component {
   }
 
   renderTextFieldAndInput(formKey, inputName, labelMessage) {
-    return this.renderTextFieldOrAreaAndInput(
-        formKey, inputName, labelMessage, false);
+    return (
+        <TextField
+          label={this.props.intl.formatMessage(labelMessage)}
+          outlined
+        >
+          <Input
+            name={inputName}
+            value={this.state[formKey]}
+            onChange={(e) => this.setState({[formKey]: e.target.value})} />
+        </TextField>
```

Comment:
might be worth extracting a renderInput method, to deduplicate? i'm not sure, up to you.

Reply:
I mean, it's four lines, so I think I'm just going to leave it.